client/packages/coldchain/src/Equipment/ListView/ListView.tsx

```diff
@@ -52,7 +53,7 @@ const AssetListComponent: FC = () => {
 
   const columnsToCreate: ColumnDescription<AssetRowFragment>[] = [];
   if (isCentralServer)
-    columnsToCreate.push({
+    columnsToCreate.push(GenericColumnKey.Selection, {
```

Comment:
This can just be 'selection', if you check `ColumnDefinitionSetBuilder.ts` you can see a list of predefined columns (: